invenio_app_ils/circulation/loaders/schemas/json/base.py

```diff
@@ -9,11 +9,23 @@
 
 from flask import current_app
 from flask_babelex import lazy_gettext as _
-from invenio_records_rest.schemas import RecordMetadataSchemaJSONV1
+from invenio_rest.serializer import BaseSchema as InvenioBaseSchema
```

Comment:
Why not `BaseSchema`?

Reply:
because `Schema` is coming from `marshmallow`, and for me `BaseSchema` might mean that it is also coming from there. I want to make sure that we understand that it is coming from Invenio, for some reasons.